web/ViewModels/LandlordProfileModel.cs

```diff
@@ -60,10 +60,9 @@ public class LandlordProfileModel
 

```

Comment:
So I think it may be worth adding the list of possible titles to this model, instead of having it as a magic string. We can then loop over that list in the view for the options, and have it here for the check on line 89 as well